static/js/showDetailTable.js

```diff
@@ -5,6 +5,118 @@
 
 // TODO, rename
 var lastSelectCaseno = undefined;
+var table = undefined;  // google table view
+var data = undefined;   // google data 
+
+function createAccData(acc_list){
+  var data = new google.visualization.DataTable();
+  name_list = ['Caseno', 'date', 'time', 'NumK', 'NumA', 'NumB','NumC'];
+  type_list = ['number', 'string', 'string', 'number', 'number', 'number', 'number'];
+
+  for( var i = 0;i < name_list.length;i++){
+    data.addColumn(type_list[i], name_list[i]);
+  }
+
+  for(var i = 0;i < acc_list.length;i++){
+    data.addRow([
+        acc_list[i].caseno,
```

Comment:
In my new code, this section has been updated with more parameters. Please check showDetailTable.js line 35-68.